include/freerdp/window.h

```diff
@@ -72,6 +72,9 @@
 #define WINDOW_ORDER_FIELD_DESKTOP_ZORDER          0x00000010
 #define WINDOW_ORDER_FIELD_DESKTOP_ACTIVE_WND      0x00000020
 
+extern const UINT32 WINDOW_ORDER_VALUES[];
```

Comment:
No need to export these, only used internally and if necessary wrap it in a function.

Reply:
Ok.